.gitignore

```diff
@@ -1,3 +1,5 @@
+build/
+.gradle/
```

Comment:
das gehört eigentlich in dein globales gitignore, schließlich muss es in jedem Projekt ignoriert werden ^^

Reply:
Naja, es betrifft jeden, der das Repo nutzen möchte, daher gehört es IMO definitiv auch in dieses gitignore.